yarn-project/sequencer-client/src/sequencer/public_processor.ts

```diff
@@ -211,7 +214,7 @@ export class PublicProcessor {
     const unencryptedLogsHash = to2Fields(result.unencryptedLogs.hash());
     const unencryptedLogPreimagesLength = new Fr(result.unencryptedLogs.getSerializedLength());
 
-    return PublicCircuitPublicInputs.from({
+    const pub = PublicCircuitPublicInputs.from({
```

Comment:
Seems like you printed `pub` for debugging, but as not done anymore, you might as well return it directly here 👍.